updates/location_asset_mappings/v6.json

```diff
@@ -0,0 +1,254 @@
+      }, {
+        "asset": "eip155:1/erc20:0x808507121B80c02388fAd14726482e061B8da827",
+        "location": "okx",
+        "location_symbol": "PENDLE"
+      }, {
+        "asset": "eip155:1/erc20:0x58D97B57BB95320F9a05dC918Aef65434969c2B2",
+        "location": null,
+        "location_symbol": "MORPHO"
+      }, {
+        "asset": "eip155:534352/erc20:0xd29687c813D741E2F938F4aC377128810E217b1b",
+        "location": null,
+        "location_symbol": "SCR"
+      }, {
+        "asset": "RLUSD",
+        "location": "independentreserve",
+        "location_symbol": "Rlusd"
+      }, {
+        "asset": "eip155:1/erc20:0x45804880De22913dAFE09f4980848ECE6EcbAf78",
+        "location": "kraken",
+        "location_symbol": "PAXG"
+      }, {
+        "asset": "eip155:1/erc20:0x157a6df6B74F4E5E45af4E4615FDe7B49225a662",
+        "location": "woo",
+        "location_symbol": "ISLAND"
+      }, {
+        "asset": "eip155:1/erc20:0x44ff8620b8cA30902395A7bD3F2407e1A091BF73",
+        "location": null,
+        "location_symbol": "VIRTUAL"
+      }, {
+        "asset": "eip155:1/erc20:0x812Ba41e071C7b7fA4EBcFB62dF5F45f6fA853Ee",
+        "location": null,
+        "location_symbol": "NEIRO"
+      }, {
+        "asset": "eip155:1/erc20:0x226bb599a12C826476e3A771454697EA52E9E220",
+        "location": null,
+        "location_symbol": "PRO"
+      }, {
+        "asset": "BLAST",
+        "location": null,
+        "location_symbol": "BLAST"
+      }, {
+        "asset": "GYEN",
+        "location": null,
+        "location_symbol": "GYEN"
+      }, {
+        "asset": "eip155:1/erc20:0xd0a6053f087E87a25dC60701ba6E663b1a548E85",
+        "location": null,
+        "location_symbol": "LRDS"
+      }, {
+        "asset": "PENGU",
+        "location": null,
+        "location_symbol": "PENGU"
+      }, {
+        "asset": "HMSTR",
+        "location": null,
+        "location_symbol": "HMSTR"
+      }, {
+        "asset": "CHILLGUY",
+        "location": null,
+        "location_symbol": "CHILLGUY"
+      }, {
+        "asset": "MICHI",
+        "location": null,
+        "location_symbol": "MICHI"
+      }, {
+        "asset": "FARTCOIN",
+        "location": null,
+        "location_symbol": "FARTCOIN"
+      }, {
+        "asset": "PNUT",
+        "location": null,
+        "location_symbol": "PNUT"
+      }, {
+        "asset": "eip155:8453/erc20:0x4ed4E862860beD51a9570b96d89aF5E1B0Efefed",
+        "location": null,
+        "location_symbol": "DEGEN"
+      }, {
+        "asset": "GBP",
+        "location": "binance",
+        "location_symbol": "BGBP"
+      }, {
+        "asset": "USD",
+        "location": "binance",
+        "location_symbol": "TUSDB"
+      }, {
+        "asset": "NGN",
+        "location": null,
+        "location_symbol": "NGN"
+      }
+    ],
+    "updates": [
+      {
+        "asset": "eip155:1/erc20:0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
+        "location": null,
+        "location_symbol": "COW"
```

Comment:
why does the cow need an update?

Reply:
it's used by both kucoin and coinbase that's why i set it to `null`.